scripts/smart-dump/defaultRecipe.js

```diff
@@ -2,12 +2,12 @@ const entries = [
   {
```

Comment:
The file name should ideally be kebab case (https://github.com/opencollective/opencollective/blob/main/rfcs/004-file-naming-convention.md#2-file-names-should-be-kebab-case).

It would also be great to have 2 separate recipes:
1. `host-admin-recipe.js`
2. `collective-admin-recipe.js`

Eventually importing one into the other if we want to combine the logic.